examples/tf_mpi/README.md

```diff
@@ -0,0 +1,33 @@
+# MNIST with TensorFlow using MPI through Horovod
+
+This little example show you how to do some distributed training using TensorFlow and Horovod.
```

Comment:
Could you explain the interest of usin Horovod over say [`tf.distribute.MirroredStrategy`](https://www.tensorflow.org/guide/distributed_training#mirroredstrategy). I would myself be very interested and I guess it will be of interest to others.

Reply:
I don't know, and I would also be interested.

Reply:
I think the difference, from what I understand re-reading your PR, is that Horovod allows you to work in a multi-node setting whereas the `tf.distribute.MirroredStrategy` only allows to work on a single node, at least out of the box.